Tensile/KernelWriterBetaOnly.py

```diff
@@ -58,7 +58,10 @@ def functionSignature(self):
 
     # pointers
     ptrStr = self.state["ProblemType"]["DestDataType"].toDevice(self.language)
-    ptrStr = "float" if self.state["_GlobalAccumulation"] else ptrStr
+    if self.state["_GlobalAccumulation"]:
+      ptrStr = self.state["ProblemType"]["ComputeDataType"].toDevice(self.language)
+      if self.state["ProblemType"]["DataType"].isHalf() and self.state["ProblemType"]["HighPrecisionAccumulate"]:
+        ptrStr = "float"
```

Comment:
Does this include bf16?

Reply:
Definition of HPA FP16 is wrong.
Compute type of HPA FP16 should be float.
That is why we only modify it back to float.
Compute type of BF16 is float.